src/Components/SmartComponents/CVEDetailsPage/CveDetailTableTitle.test.js

```diff
@@ -0,0 +1,68 @@
+import '@testing-library/jest-dom';
+import { render, screen, act, waitFor, within } from '@testing-library/react';
+import { getAffectedSystemsByCVE } from '../../../Helpers/APIHelper';
+import CVEDetailTableTitle from './CVEDetailTableTitle';
+
+jest.mock('@unleash/proxy-client-react', () => ( {
+    ...jest.requireActual('@unleash/proxy-client-react'),
+    useFlag: () => true, 
+    useFlagsStatus: () => ({ flagsReady: true })
+}))
+
+jest.mock('../../../Helpers/APIHelper', () => ({
+    ...jest.requireActual('../../../Helpers/APIHelper'),
+    getAffectedSystemsByCVE: jest.fn()
+}));
+
+afterEach(() => {
+    jest.resetAllMocks()
+});
+
+getAffectedSystemsByCVE.mockImplementation(() => () => Promise.resolve({}))
+
+const waitComponent = async () => {
+    await act( async () => {
+        render(<CVEDetailTableTitle cveName={'test-cve'}  />);
+    });
+    await waitFor(() => {
+        expect(screen.getByLabelText('Affected systems table title')).toBeDefined();
```

Comment:
```suggestion
        expect(screen.getByLabelText('Affected systems table title')).toBeVisible();
```

TBH, never saw toBeDefined. I think the standard way is to check for visibility

Reply:
It is fine to use both: https://testing-library.com/docs/preact-testing-library/example/

Reply:
Yeah, but if an element is in DOM it doesn't necessarily mean that it is visible.